### DISCUSSION TITLE
No Sound with Gotek and Atari Ste

### DISCUSSION THREAD

**Discussion by momone50** [keirf/flashfloppy#450] [Q&A] [answered]
Hi, i attached a Buzzer on JB pins as explained, and i don't have any sound.
I assumed my buzzer what out of order, so i decided to attach a magnetic speaker with npn transistor and a 1k resistor, as explained in the wiki. I don't have sound. I used a 2n2222 transistor, as i didn't have a spare 2N3904 but should be equivalent enough. No?
I tested it with an arduino and it works great...

Did i missed something, like a jumper setting, or wathever? 
Maybe it needs a special version of flashfloppy? 
Maybe the gotek doesn't supply  enough current to drive my transistor?

Thanks for your help

**Comment by keirf** [comment]
There is no jumper setting, you simply need to connect across JB (as you did). The buffer circuit with 2n2222 should also work fine.

Perhaps header pins are badly soldered at JB on your Gotek? Also the firmware only outputs a short pulse on each head step. Nonetheless this should be quite audible as a few 100Hz tone when the heads seek long distances.

**Reply by momone50** [reply]
It doesn't "reproduce" floppy sounds as with HCX? just  small pulses
?

**Reply by keirf** [reply]
It's the same as HxC actually, I believe. The small pulses actually emulate drive sound okay. Pretty much as well as can be achieved with a tinny sounder, anyway.

**Comment by momone50** [comment, marked as answer]
Just tried with another piezo i had in stock. It works, even if sound is really low. I would have prefered using an HP and Transistor but i don't know why it just doesn't work...
I mark as answered
